ext/RastersArchGDALExt/cellsize.jl

```diff
@@ -50,7 +50,8 @@ end
 
 function cellsize(dims::Tuple{X, Y})
```

Comment:
Does this dispatch even work? that seems pretty limited

Reply:
If you call cellsize on a raster it just takes the X and Y dimension out. But ideally this would be defined for any Tuple of Dimensions, I think. I'm just not sure how to dispatch on that?

Reply:
But shouldn't it be `Tuple{<:X,<:Y}` at least? they have a type parameter

Reply:
You can just use `DimTuple` for dispatch. But then you need to choose dims by name with `DD.dims(dims, X)`

Reply:
I now changed this to `Tuple{<:X,<:Y}` and added a dispatch for `DimTuple` separately.